test/test_utils_config_module.py

```diff
@@ -155,6 +159,7 @@ def test_save_config_portable(self):
                 "e_env_default_FALSE": False,
                 "e_env_force": True,
                 "e_optional": True,
+                "e_aliased_bool": False,
```

Comment:
Shouldn't this not get recorded? (Due to the check for if it's aliased in the core code?).

Reply:
Code is saying it will not get set again, nothing about recording?

Reply:
Ah sorry I misread some code, ignore me. (I thought your alias suppression was on saving config not loading configs).